src/container/plugins/RemotePlugin.js

```diff
@@ -32,50 +31,54 @@
 		this.openRemote = function(api, options, playOptions)
 		{
 			// This should be deprecated, support for old function signature
-			if (typeof options === "boolean")
+			if (typeof options === 'boolean')
 			{
 				options = {
 					singlePlay: singlePlay,
 					playOptions: playOptions
 				};
 			}
-			options = $.extend(
-			{
-				query: '',
-				playOptions: null,
-				singlePlay: false
-			}, options);
+			options = Object.assign(
+				{
+					query: '',
+					playOptions: null,
+					singlePlay: false
+				},
+				options
+			);
 
 			this.release = null;
 
-			$.getJSON(api, function(result)
+			fetch(api)
```

Comment:
Alas, [fetch is not supported in IE 11](https://caniuse.com/#feat=fetch), so we'll probably just have to do an XHR request here with an `onload` callback ([`Promise` isn't supported either](https://caniuse.com/#feat=promises))